_tests/integration/docker_create_bitrise.yml

```
@@ -0,0 +1,45 @@
+format_version: 1.3.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+workflows:
+  docker-create-fails-invalid-port:
+    title: Expected to fail on docker create, invalid port provided
+    container:
+      image: frolvlad/alpine-bash:latest
+      ports:
+      - 22:22
+    steps:
+    - script:
+        title: Should fail
```

```suggestion
        title: Successful step
```

This confused me for a second, but I agree that the step itself should return 0, so that the test verifies the correct error!